### PR TITLE
Impove PHPStan workflow

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see comments below.

### CODE REVIEW

File: .github/workflows/phpstan.yml [hunk 1]
```diff
@@ -22,8 +22,8 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  composer require "php-ffmpeg/php-ffmpeg" --no-interaction --no-update
-                  composer update --prefer-dist --no-interaction
+                  composer require --no-interaction --no-update php-ffmpeg/php-ffmpeg
+                  composer update --no-interaction --prefer-dist
 
             - name: Run PHPStan
-              run: ./vendor/bin/phpstan --error-format=github
```
Comment (szepeviktor) [inline .github/workflows/phpstan.yml:29]: This is automatic 🎉

File: .github/workflows/phpstan.yml [hunk 2]
```diff
@@ -22,8 +22,8 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  composer require "php-ffmpeg/php-ffmpeg" --no-interaction --no-update
-                  composer update --prefer-dist --no-interaction
+                  composer require --no-interaction --no-update php-ffmpeg/php-ffmpeg
+                  composer update --no-interaction --prefer-dist
```
Comment (szepeviktor) [inline .github/workflows/phpstan.yml:26]: Use Composer options as documentation suggests.

File: .github/workflows/phpstan.yml [hunk 3]
```diff
@@ -22,8 +22,8 @@ jobs:
...
 
             - name: Run PHPStan
-              run: ./vendor/bin/phpstan --error-format=github
+              run: ./vendor/bin/phpstan
```
Comment (szepeviktor) [inline .github/workflows/phpstan.yml:29]: This is automatic 🎉

**Comment by freekmurze** [comment]
Thanks!

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**